platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -365,7 +386,7 @@ protected boolean doLogin() throws LoginException {
                         }
                     }
                 } catch (Exception e) {
-                    boolean result;
+                    LOGGER.debug("Exception while getting roles for user.", e);
```

Comment:
❓ Should we add the user name and some more information to this log since that information won't show up in the logs unless the logging level is set to `trace` (L367-372)?